modules/installation-aws-permissions.adoc

```diff
@@ -154,7 +154,9 @@ If you use an existing Virtual Private Cloud (VPC), your account does not requir
 
 [NOTE]
 =====
-If you have not created a load balancer in your AWS account, the IAM user also requires the `iam:CreateServiceLinkedRole` permission.
+* If you specify an existing IAM role in the `install-config.yaml` file, the following IAM permissions are not required: `iam:CreateRole`,`iam:DeleteRole`, `iam:DeleteRolePolicy`, and `iam:PutRolePolicy`.
+
+* If you have not created a load balancer in your AWS account, the IAM user also requires the `iam:CreateServiceLinkedRole` permission.
```

Comment:
@brendan-daly-red-hat why did we put LB thing here, is it related to the issue?

Reply:
Hi @yunjiang29, The text about the load balancer ('If you have not created a load balancer...permission.') is an existing note (see [Required AWS permissions for the IAM user](https://docs.openshift.com/container-platform/4.16/installing/installing_aws/installing-aws-account.html#installation-aws-permissions_installing-aws-account)). Rather than add a separate 'Note' for this PR text, I added the text to the same note as a bulleted list.

Reply:
ah, got it, thanks